src/App.tsx

```diff
@@ -31,6 +31,8 @@ class App extends React.Component<Props, State>  {
           <Route path="/videos/:series" render={props => <HomePage pageType="video" {...props} />} />
           <Route path="/posts/:blog" render={props => <HomePage pageType="blog" {...props} />} />
           <Route path="/notes/:note" render={props => <HomePage pageType="note" {...props} />} />
+          <Route path="/video-archive/:videoType" render={props => <HomePage pageType="video-archive" {...props} />} />
```

Comment:
What about `/archive/video` and `/archive/series`. You can add another `Switch`/`Route` to `Archive` to change the logic for archive and series, like I did with notes. The code will end up being cleaner if we give some hierarchy to the routing. We are essentially using `pageType` to duplicate the information that's in the URL, we could just be using that.

Reply:
To clarify, you mean `/archive/video/:foo` and `/archive/series/:foo`, right?